examples/getting-started/hello-instancing-low/app.js

```diff
@@ -1,4 +1,5 @@
 /* global window, requestAnimationFrame, cancelAnimationFrame */
+import 'core-js';
```

Comment:
I’d suggest importing this once in the website template.

Reply:
I want the examples to run on their own in IE and Edge. Is it possible to do that via the HTML webpack plugin or something like that?

Reply:
I believe that’s what https://babeljs.io/docs/en/babel-preset-env#usebuiltins is for but we’ve had issues with it...